### PR TITLE
fix duplicated IDs

### PR DESCRIPTION
**Pull request by MaruHyl**

### CODE REVIEW

File: pool.go [hunk 1]
```diff
@@ -232,12 +233,20 @@ func (pool *Pool) stopSupervisor() {
 // adds it to the Pool.
 func (pool *Pool) Add(f func(...interface{}) interface{}, args ...interface{}) {
 	job := &Job{f, args, nil, nil, make(chan bool), 0, pool.Next_job_id}
```
Comment (stefantalpalaru) [inline pool.go]: Did you forget to replace pool.Next_job_id with pool.getNextJobId() here?
This also shows that you're testing the wrong thing in TestGetNextJobId(). We care about actual jobs having unique IDs, not sync/atomic doing what it's supposed to do.

Comment (MaruHyl) [reply pool.go]: yes,i forget..  now what should i do? close this pr?

Comment (stefantalpalaru) [reply pool.go]: No, just push more commits to your branch and they will appear here.

Comment (MaruHyl) [reply pool.go]: i pushed a commit a moment ago

**Comment by stefantalpalaru** [comment]
Thank you for your contribution. I had to refactor it for reasons explained here: https://github.com/stefantalpalaru/pool/commit/d49ee1c57afa54257eede9251e06c2f6b8e15019

**Pull request merged**